ts/indicators/indicators.src.ts

```diff
@@ -42,8 +42,8 @@ declare global {
             public getName(): string;
             public getValues(
                 series: Series,
-                params: object
-            ): IndicatorValuesObject;
+                params: SmaIndicatorParamsOptions
+            ): (boolean|IndicatorValuesObject);
```

Comment:
@bre1470 
What is the reason of setting here a boolean type? Because in all cases it returns array of points (empty array)

Reply:
Inherited indicators might return boolean and you can not change a return type between related classes.

Reply:
I do not know any scenario when getValues can be boolean, could you elaborate how you are able reproduce that?

Reply:
Basically when there are less values available on the referenced series than the period param requires: https://github.com/highcharts/highcharts/pull/11835/files#diff-bb3b226e917ae389d52c66c3939658ddR186-R188